iambic/plugins/v0_1_0/aws/cloud_formation/utils.py

```diff
@@ -429,7 +472,27 @@ async def create_iambic_role_stacks(
             f"https://{region}.console.aws.amazon.com/cloudformation/home?region={region}#/stacksets/IambicSpokeRole/stacks\n"
         )
         return await create_spoke_role_stack_set(
-            cf_client, org_client, hub_account_id, read_only=read_only
+            cf_client,
+            org_client,
+            hub_account_id,
+            read_only=read_only,
+            stack_set_name=spoke_role_stack_name,
+            hub_role_name=hub_role_name,
+            spoke_role_name=spoke_role_name,
+            tags=tags,
         )
 
     return hub_role_created
+
+
+def parse_pair(s) -> dict:
+    key, value = s.rstrip().split("=")
+    return {"Key": key, "Value": value}
```

Comment:
Do we need to do some regex validation on tags? And char limit check? (I feel like we have this type of function somewhere in cloudumi but I can't find it)

Reply:
added validate callback in questionnaire. I check aws-cli doesn't actually check on the cli level, and defer regex check on boto3.